Author: Jamie Strandboge <jamie@canonical.com>
Description: aa-status was written to work with python 2 or 3. Upstream is
 still using 2, so adjust ours to use /usr/bin/python3 to avoid pulling
 python 2 back to the desktop images
Forwarded: no
---
 utils/aa-audit      |    2 +-
 utils/aa-autodep    |    2 +-
 utils/aa-cleanprof  |    2 +-
 utils/aa-complain   |    2 +-
 utils/aa-disable    |    2 +-
 utils/aa-enforce    |    2 +-
 utils/aa-genprof    |    2 +-
 utils/aa-logprof    |    2 +-
 utils/aa-mergeprof  |    2 +-
 utils/aa-status     |    2 +-
 utils/aa-unconfined |    2 +-
 11 files changed, 11 insertions(+), 11 deletions(-)

Index: apparmor-2.10.95/utils/aa-status
===================================================================
--- apparmor-2.10.95.orig/utils/aa-status
+++ apparmor-2.10.95/utils/aa-status
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ------------------------------------------------------------------
 #
 #    Copyright (C) 2005-2006 Novell/SUSE
Index: apparmor-2.10.95/utils/aa-audit
===================================================================
--- apparmor-2.10.95.orig/utils/aa-audit
+++ apparmor-2.10.95/utils/aa-audit
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-autodep
===================================================================
--- apparmor-2.10.95.orig/utils/aa-autodep
+++ apparmor-2.10.95/utils/aa-autodep
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-cleanprof
===================================================================
--- apparmor-2.10.95.orig/utils/aa-cleanprof
+++ apparmor-2.10.95/utils/aa-cleanprof
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-complain
===================================================================
--- apparmor-2.10.95.orig/utils/aa-complain
+++ apparmor-2.10.95/utils/aa-complain
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-disable
===================================================================
--- apparmor-2.10.95.orig/utils/aa-disable
+++ apparmor-2.10.95/utils/aa-disable
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-enforce
===================================================================
--- apparmor-2.10.95.orig/utils/aa-enforce
+++ apparmor-2.10.95/utils/aa-enforce
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-genprof
===================================================================
--- apparmor-2.10.95.orig/utils/aa-genprof
+++ apparmor-2.10.95/utils/aa-genprof
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-logprof
===================================================================
--- apparmor-2.10.95.orig/utils/aa-logprof
+++ apparmor-2.10.95/utils/aa-logprof
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
Index: apparmor-2.10.95/utils/aa-mergeprof
===================================================================
--- apparmor-2.10.95.orig/utils/aa-mergeprof
+++ apparmor-2.10.95/utils/aa-mergeprof
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #    Copyright (C) 2014-2015 Christian Boltz <apparmor@cboltz.de>
Index: apparmor-2.10.95/utils/aa-unconfined
===================================================================
--- apparmor-2.10.95.orig/utils/aa-unconfined
+++ apparmor-2.10.95/utils/aa-unconfined
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 # ----------------------------------------------------------------------
 #    Copyright (C) 2013 Kshitij Gupta <kgupta8592@gmail.com>
 #
